shared/lib/token-utils.js

```diff
@@ -55,18 +64,18 @@ export const isTokenVerified = (tokenAddress, networkType) =>
     ? ETHER_TOKEN_VERIFIED_ADDRESSES.has(tokenAddress.toLowerCase())
     : true
 
-export const tokenDataFallback = (tokenAddress, fieldName, networkType) => {
+export const tokenDataOverride = (tokenAddress, fieldName, networkType) => {
   // The fallback list is without checksums
```

Comment:
Should we change the "fallback" in this comment, too? It seems we're not using fallbacks anymore.